airbyte-integrations/connectors/source-recharge/source_recharge/manifest.yaml

```diff
@@ -0,0 +1,269 @@
+    inject_into:
+      type: RequestOption
+      inject_into: header
+      field_name: X-Recharge-Access-Token
+
+  # PAGINATORS
+  paginator_deprecated_api:
+    type: DefaultPaginator
+    page_token_option:
+      type: RequestOption
+      inject_into: request_parameter
+      field_name: page
+    page_size_option:
+      inject_into: request_parameter
+      field_name: limit
+      type: RequestOption
+    pagination_strategy:
+      type: PageIncrement
+      start_from_page: 1
+      page_size: 250
+      inject_on_first_request: false
+  paginator_modern_api:
+    type: DefaultPaginator
+    page_token_option:
+      type: RequestOption
+      inject_into: request_parameter
+      field_name: cursor
+    page_size_option:
+      inject_into: request_parameter
+      type: RequestOption
+      field_name: limit
+    pagination_strategy:
+      type: CursorPagination
+      page_size: 250
+      cursor_value: '{{ response.get("next_cursor", {}) }}'
+      stop_condition: '{{ not response.get("next_cursor", {}) }}'
+
+  # REQUESTERS
+  requester_base:
+    description: >-
+      Default Base Requester for Full Refresh streams
+    type: HttpRequester
+    url_base: https://api.rechargeapps.com/
+    path: "{{ parameters['name'] }}"
+    http_method: GET
+    authenticator:
+      $ref: "#/definitions/authenticator"
+    error_handler:
+      type: DefaultErrorHandler
+      description: >-
+        The default error handler
+  requester_deprecated_api:
+    $ref: "#/definitions/requester_base"
+    # for deprecated retriever we should use `2021-01` api version
+    request_headers:
+      x-recharge-version: "2021-01"
+  requester_modern_api:
+    $ref: "#/definitions/requester_base"
+    # for modern retriever we should use >= `2021-11` api version
+    request_headers:
+      x-recharge-version: "2021-11"
+
+  # RETRIEVER FOR `DEPRECATED API`
+  retriever_api_deprecated:
+    description: >-
+      Default Retriever for Deprecated API `2021-01` Full Refresh streams.
+    record_selector:
+      $ref: "#/definitions/selector"
+    requester:
+      $ref: "#/definitions/requester_deprecated_api"
+    paginator:
+      $ref: "#/definitions/paginator_deprecated_api"
+  # RETRIEVER FOR `MODERN API`
+  retriever_api_modern:
+    description: >-
+      Default Retriever for Modern API `2021-11` Full Refresh streams.
+    record_selector:
+      $ref: "#/definitions/selector"
+    requester:
+      $ref: "#/definitions/requester_modern_api"
+    paginator:
+      $ref: "#/definitions/paginator_modern_api"
+    # we should ignore all other req.params once we have the `next_page_token` in response
+    # for pagination in `2021-11` - modern api.
+    ignore_stream_slicer_parameters_on_paginated_requests: true
+  # RETRIEVER FOR `METAFIELDS` STREAM
+  retriever_metafields:
+    $ref: "#/definitions/retriever_api_modern"
+    partition_router:
+      type: ListPartitionRouter
+      cursor_field: owner_resource
+      values:
```

Comment:
The previous version would only fetch for customer, store and subscription. I don't know enough to confirm but it would seem like this is a valid change so I'm fine with it